### DISCUSSION TITLE
Move logo link to the logo partial

### DISCUSSION THREAD

**Discussion by pepito2k** [avo-hq/avo#680] [Feature requests]
Right now, when ejecting the logo partial, the partial only contains the image, and there are styles applied to the link tag that may affect the image being displayed properly.

I know this can be solved by ejecting the sidebar and customizing it. But it seems like a lot to do just to change a css class name in the link that forms part of the logo being displayed.

current `avo/app/views/avo/partials/_logo.html.erb`:

``` 
<%= image_tag '/avo-assets/logo.png', class: 'h-full', title: 'Avo' %>
```

proposed partial:
```
<%= link_to root_path, class: 'logo-placeholder h-16 bg-white p-2 flex justify-center' do %>
  <%= image_tag '/avo-assets/logo.png', class: 'h-full', title: 'Avo' %>
<% end %>
```

**Comment by adrianthedev** [comment]
That makes sense. Let me try to squeeze this in the next release

**Comment by adrianthedev** [comment]
Done via https://github.com/avo-hq/avo/pull/683.

I'll cut a release on Monday. Have an amazing weekend!
